### DISCUSSION TITLE
Feature request: Note hierarchy

### DISCUSSION THREAD

**Discussion by ashishbhateja** [notaapp/talk#216] [Feature Requests]
Support to hierarchy in notes, e.g. [[A]]/[[B]]/[[C]]

**Comment by astoilkov** [comment]
Hmm. Can you provide a little more information? Have you seen this feature in other products?

I can think of Logseq [[A/B/C]] wiki-links. However, we have folders and that could be achieved with folders (you can specify the full path instead of the name of the file — [[folder/folder/file]] instead of [[file]]).

In order to understand your idea better, you can say how you expect Nota to behave differently if you have a link like [[A]]/[[B]]/[[C]]?

Thanks in advance!

**Comment by ashishbhateja** [comment]
Thank you for the revert. In my view, the folder structure is different compared to the note hierarchy. For example, I have my fleeting notes located in a folder called “Fleeting”. Each of them may belong to one or more hierarchies. It’s easier to do a soft link using square brackets than search in the folder. Also, if the hierarchy is visible on the note (breadcrumb in Obsidian or hierarchy in Logseq), the context is shown along with the back links.

Also, in the example above, [[A]] and [[B]] can be notes while still be part of the hierarchy.

I can use the folder path, but that’s pretty much similar using the file path instead of wikilinks. It’s a rigid structure vs thought flow. 

I hope this helps.

**Comment by fedorovpishet** [comment]
Its look like notes branches and sequences in Zettelkasten. I've saw something similar in Scrivener and Supernotes.

**Comment by astoilkov** [comment]
I've started researching this because I believe it has a lot of potential. I will post with an overview and suggestions on what we can do. I can also have some questions, I hope you don't mind.
